can_tools/scrapers/official/IA/__init__.py

```diff
@@ -0,0 +1 @@
+from can_tools.scrapers.official.IA.ia_vaccine_county import IowaCountyVaccine
```

Comment:
I did a big cleanup to reduce boilerplate and not put stuff in these `__init__.py` files

Let's delete the content of the file

The file must exist to make the other code `import`able, but let's leave it empty

Reply:
Done